pkgdb/build-aux/changed-sources

```diff
@@ -35,120 +34,125 @@ OPTIONS
 
 ENVIRONMENT
   GIT               Command used as \`git' executable.
-";
-
+"
 
 # ---------------------------------------------------------------------------- #
 
 usage() {
-  if [[ "${1:-}" = "-f" ]]; then
-    echo "$_help_msg";
-  else
-    echo "$_usage_msg";
-  fi
+	if [[ "${1:-}" = "-f" ]]; then
+		echo "$_help_msg"
```

Comment:
Suggestion: It looks like we need an indentation rule for scripts that don't end in `.sh`, this looks like it's using 4 spaces instead of 2.

Reply:
this file is not recognized as a shell file (no .sh/.bash extension)